src/remote-data-t.ts

```diff
@@ -2,44 +2,46 @@ import * as RD from './remote-data';
 import { RemoteData, remoteData, URI } from './remote-data';
 
 import {
-	ApplicativeComposition,
 	ApplicativeComposition12,
 	ApplicativeComposition22,
 	getApplicativeComposition,
 	Applicative2,
 	Applicative1,
 	Applicative,
+	ApplicativeCompositionHKT2,
 } from 'fp-ts/lib/Applicative';
-import { HKT, Type, Type2, URIS, URIS2 } from 'fp-ts/lib/HKT';
+import { HKT, Kind, Kind2, URIS, URIS2 } from 'fp-ts/lib/HKT';
 import { Monad, Monad1, Monad2 } from 'fp-ts/lib/Monad';
 import { Functor2, Functor1, Functor } from 'fp-ts/lib/Functor';
-import { Function1, Lazy } from 'fp-ts/lib/function';
+import { FunctionN, Lazy } from 'fp-ts/lib/function';
 
-export interface RemoteDataT<F> extends ApplicativeComposition<F, URI> {
+export interface RemoteDataT<M, E, A> extends HKT<M, RemoteData<E, A>> {}
+
+export interface RemoteDataM<F> extends ApplicativeCompositionHKT2<F, URI> {
```

Comment:
@YBogomolov Could you help with reviewing the transformer module please?

Reply:
Looks good to me. I like how the code style is streamlined with fp-ts@2 style 👍🏻